more/free-programming-cheatsheets.md

```diff
@@ -398,6 +398,7 @@
 ### Unit testing
 
 * [Chai CheatSheet](https://devhints.io/chai) - devhints, Rico Santa Cruz (HTML)
+* [Cypress CheatSheet](https://cheatography.com/aiqbal/cheat-sheets/cypress-io/) - aiqbal (HTML)
```

Comment:
Hey @Hari70a try to put all the resources in a single PR instead of raising multiple PRs with one resource each.

Reply:
Ok @shrn01 will do like that hereafter